.github/workflows/check-metabolictasks.yml

```
@@ -0,0 +1,27 @@
+name: Test metabolic tasks
+
+on: [push, pull_request]
+
+jobs:
+  check-metabolictasks:
+    runs-on: self-hosted
+    timeout-minutes: 60
+
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Check essential tasks
+      run: |
+        /usr/local/bin/matlab -nodisplay -nosplash -nodesktop -r "addpath(genpath('.')); run('code/test/testMetabolicTasks.m', '${{ matrix.param }}'); exit;"
+      env:
+        matrix:
+          param: 'essential'
+
+    - name: Check verification tasks
+      run: |
+        /usr/local/bin/matlab -nodisplay -nosplash -nodesktop -r "addpath(genpath('.')); run('code/test/testMetabolicTasks.m', '${{ matrix.param }}'); exit;"
+      env:
+        matrix:
+          param: 'verification'
```

I guess the purpose here was to [use a matrix like in the docs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#using-a-matrix-strategy):
```suggestion
          param: [essential, verification]
```